app/talk/popular-tags.cjsx

```diff
@@ -23,6 +26,7 @@ module?.exports = React.createClass
     talkClient.type('tags/popular').get query
 
   tag: (talkTag, i) ->
+    logClick = @context.geordi?.makeHandler? 'hashtag-sidebar'
```

Comment:
@wgranger can you check this? delete if not being used, confirm used and we can ignore this comment if it is.